src/SqlConverters.cs

```diff
@@ -162,8 +162,10 @@ public virtual async Task<string> BuildItemFromAttributeAsync(SqlAttribute attri
                 {
                     adapter.SelectCommand = command;
                     await connection.OpenAsync();
+                    Dictionary<TelemetryPropertyName, string> props = connection.AsConnectionProps();
                     var dataTable = new DataTable();
                     adapter.Fill(dataTable);
+                    TelemetryInstance.TrackEvent(TelemetryEventName.BuildItemFromAttributeAsync, props);
```

Comment:
Maybe we can pass the ConvertType to BuildItemFromAttributeAsync and use `TelemetryInstance.TrackConvert(type, props)` here instead of creating a new event?

Reply:
+1 - we already have an event for input bindings so we shouldn't change that.

Reply:
@MaddyDev That's what Lucy is saying. Add a parameter for the type of convert being done and then move the TraceConvert call into this method instead